lib/datadog/appsec/configuration.rb

```diff
@@ -14,7 +14,9 @@ def self.included(base)
 
       # Configuration DSL implementation
       class DSL
-        Instrument = Struct.new(:name, :options)
+        # Struct constant whisker cast for Steep
```

Comment:
What's is this `whisker` pattern here for Steep?

Reply:
Assigning to `_` swallows typing information so that `_ = foo` always works. As a consequence `bar = _ = foo` acts as a forceful cast, ignoring the type of `foo` when assigning to `bar`.

This pattern should not be used lightly but is sometimes necessary. In this case it's because `Struct.new` returns a `Struct[untyped]` but here we want the `Instrument` constant to be of type `Instrument`. [See here](https://github.com/ruby/rbs/issues/627#issuecomment-1335085271) for details.

(The name comes from the looks of this assignment: `= _ =` looks like cat whiskers)

Reply:
To be honest, this file is so big that I would just suggest just making it a regular `class` and not need any particular weird typing things :)

Reply:
Yeah, I thought the same. I'll try to improve that on future quality iterations.

Overall I'll be trying to refactor all these things requiring syntax or comment directive oddities.